tests/testthat/helper-rlang.R

```diff
@@ -90,3 +90,10 @@ skip_if_not_windows <- function() {
   system <- tolower(Sys.info()[["sysname"]])
   skip_if_not(is_string(system, "windows"), "Not on Windows")
 }
+
+arg_match_wrapper <- function(arg, ...) {
+  arg_match(arg, ...)
+}
+arg_match0_wrapper <- function(arg, values, arg_nm = "arg", ...) {
+  arg_match0(arg, values, arg_nm = arg_nm, ...)
```

Comment:
Why do we need these wrappers?

Reply:
This is just to test that the wrapper call appears in error messages.